crates/optimism/flashblocks/src/lib.rs

```diff
@@ -47,31 +53,24 @@ pub type InProgressFlashBlockRx = tokio::sync::watch::Receiver<Option<FlashBlock
 /// Groups together the three receivers that provide flashblock-related updates.
```

Comment:
we no longer have three receivers
```suggestion
/// Groups together the channels for flashblock-related updates.
```